### ISSUE TITLE
Builds are broken/out of date due possibly due to app-id change

### ISSUE THREAD

**Issue by ahayzen** [flathub/org.gnome.Nautilus#6]
What happened
- Tried to install the latest `org.gnome.Nautilus` flatpak from flathub, found that it is from 2017-07-26 16:54:28

What I expected to happen:
- For Nautilus 3.28 from Mar 19, 2018 to be installed.

I also found that the app-id in the manifest has changed [0] to `org.gnome.NautilusStableFlatpak`, this seems to be the possible cause of the failed build [1] ?

On IRC Alex has suggested that the repo might need to be renamed.
```irc
<ahayzen> hi, i was going to have a look at the nautilus flatpak, but i noticed that the version in the repo is super old compared to the manifest - appears a build has been missed/failed? Also in the manifest the appid has changed to org.gnome.NautilusStableFlatpak, i wonder if this has broken things? https://github.com/flathub/org.gnome.Nautilus/blob/master/org.gnome.Nautilus.json and this is the latest build log i could find https://flathub.org/builds/#/builders/2/builds/220
<alexlarsson> Yeah, that will not work..
<alexlarsson> I guess we need to rename the repo, etc
```

0 - https://github.com/flathub/org.gnome.Nautilus/blob/master/org.gnome.Nautilus.json#L2
1 - https://flathub.org/builds/#/builders/2/builds/220

**Comment by TingPing** [comment]
I think we agreed to just remove it from flathub? @csoriano1618 ?

**Comment by csoriano1618** [comment]
I guess, yes. @ramcq had some ideas/concerns.

**Comment by jurf** [comment]
May I ask, why the rename? Aren’t branches supposed to do that?

**Comment by TingPing** [comment]
@jurf He wanted a normal package and the flatpak to be separate applications which requires a unique desktop file/dbus service/etc name.

**Comment by Ads20000** [comment]
If it's being removed from the Flathub where will it be hosted in an easy-to-install fashion? :)

**Comment by TingPing** [comment]
Nightly releases still exist in `gnome-apps-nightly`.

**Comment by Ads20000** [comment]
True! Although there's not easy-to-use way to get those (in the same vein as the Flathub UI). The easiest way is to go [here](https://git.gnome.org/browse/gnome-apps-nightly/tree/) and click on the 'plain' buttons for the relevant Flatpaks, but it's not easy to find (at all!) and it's not very pretty!

**Comment by TingPing** [comment]
Well it is sort of intended for developers.

**Comment by Ads20000** [comment]
That's true but imho it should still be fairly easy to access (it can recommend stable for users) / there should be a link to it from some GNOME (dev) docs page and I'm not sure there is?

**Comment by TingPing** [comment]
I'm sure he would be open to linking to it on the GNOME wiki for Nautilus or such.

**Comment by Ads20000** [comment]
Yes please! @csoriano1618 a link to [the .flatpakref](https://git.gnome.org/browse/gnome-apps-nightly/plain/nautilus.flatpakref) on [this page](https://wiki.gnome.org/Apps/Files/Resources) would be excellent please (and then the same for the other GNOME apps)! :)

I know the Files Flatpak is very buggy but it's still been _really_ useful for me to test bugs on the Files master (allegedly, though it hasn't been updated since 2018-03-18 08:53:15 +0000) without overwriting Ubuntu's Files 3.26 or having to build it :)

**Comment by csoriano1618** [comment]
@Ads20000 that sounds like an excellent idea.

Since you are interested, wanna go forward with it? Create an account in the wiki, let me know your user and I'll give to you permissions to edit the GNOME wiki. Then you can go over those apps and add a link to the flatpakref. Reach me on IRC at #gnome-hackers as csoriano if you have any doubts/comments.

Ideally in the future would be nice to have a central entity to install dev versions for people to have installed for testing and giving feedback.

**Comment by Ads20000** [comment]
@csoriano1618 Yes I would be very happy to do that. My GNOME wiki username is Ads20000. Should I put the link under, e.g., 'Code & Bugs'? https://wiki.gnome.org/Apps/Files/Resources

Should I also link to [here](https://medium.com/@alex285/try-flatpak-gnome-files-nightly-give-feedback-2196351b6501) (for now) since it lists some known issues? Or is there another way I can incorporate that information?

**Comment by Ads20000** [comment]
Or, actually, I could put the link on the homepage like with https://wiki.gnome.org/Apps/Builder

(though Builder is hosted on Flathub...for consistency shouldn't all GNOME (non-master) apps be hosted on Flathub? Or will this all be updated soon?)

In fact, a general cleanup of the Wiki so that there's a consistent experience with accessing info for all the different GNOME apps would be good, though idk how I could get the authority and design for that.

**Comment by TingPing** [comment]
> (though Builder is hosted on Flathub...for consistency shouldn't all GNOME (non-master) apps be hosted on Flathub? Or will this all be updated soon?)

The entire point of this discussion is that Nautilus isn't a traditional app and doesn't really fit into normal usage with Flatpak so it doesn't belong on Flathub.
